engine/src/main/resources/assets/shaders/chunk_frag.glsl

```diff
@@ -112,21 +112,17 @@ void main() {
     float invMax = inversesqrt(max(dot(tangent,tangent), dot(binormal,binormal)));
     mat3 tbn = mat3(tangent * invMax, binormal * invMax, normal);
 
+#if defined (PARALLAX_MAPPING)
     vec3 eyeTangentSpace = tbn * vertexViewPos.xyz;
 
     float height =  parallaxScale * texture2D(textureAtlasHeight, texCoord).r - parallaxBias;
 	texCoord += height * normalize(eyeTangentSpace).xy * TEXTURE_OFFSET;
 #endif
 
-    normalOpaque = (texture2D(textureAtlasNormal, texCoord).xyz * 2.0 - 1.0);
-
-    // Simplified tangent basis - because we can! Voxels and blocks are great
```

Comment:
If anyone knows how to actually get the simplified tangent basis to work,  please let me know and we can switch back to something less intensive.  I did not notice any framerate difference having to compute the tbn value.